orca-core/src/main/java/com/netflix/spinnaker/orca/config/OrcaConfiguration.java

```diff
@@ -115,8 +116,8 @@ public ComparableLooseVersion comparableLooseVersion() {
   }
 
   @Bean
-  UserConfiguredUrlRestrictions userConfiguredUrlRestrictions(UserConfiguredUrlRestrictions.Builder userConfiguredUrlRestrictionProperties) {
-    return userConfiguredUrlRestrictionProperties.build();
+  UserConfiguredUrlRestrictions userConfiguredUrlRestrictions(UserConfiguredUrlRestrictions.Builder userConfiguredUrlRestrictionProperties, @Value("${urlRestrictions.rejectedIps:}") String[] rejectedIps) {
+    return userConfiguredUrlRestrictionProperties.withRejectedIps(Arrays.asList(rejectedIps)).build();
```

Comment:
You shouldn't need to manually wire in the config `urlRestrictions.rejectedIps` here.  If you use the existing configuration hash `userConfiguredUrlRestrictions` and set `userConfiguredUrlRestrictions.rejectedIps`, this will get auto-wired for you above.